Mathlib/Analysis/SpecialFunctions/Trigonometric/Bounds.lean

```diff
@@ -88,25 +88,25 @@ lemma sin_le_mul (hx : -(π / 2) ≤ x) (hx₀ : x ≤ 0) : sin x ≤ 2 / π * x
 /-- Half of **Jordan's inequality** for absolute values. -/
 lemma mul_abs_le_abs_sin (hx : |x| ≤ π / 2) : 2 / π * |x| ≤ |sin x| := by
   wlog hx₀ : 0 ≤ x
-  case inr => simpa using this (by rwa [abs_neg]) <| neg_nonneg.2 <| le_of_not_le hx₀
+  · simpa using this (by rwa [abs_neg]) <| neg_nonneg.2 <| le_of_not_le hx₀
```

Comment:
What's the justification for this change? I was under the impression that named cases are preferred to anonymous ones?

Reply:
I'm not sure that's a rule, and in any case I don't think the `inr` name from `wlog` is intentional or even a good name.

Reply:
I don't like it but that's a fair point :)